pykrige/uk.py

```diff
@@ -174,19 +175,32 @@ class UniversalKriging:
     enable_plotting : boolean, optional
         Enables plotting to display variogram. Default is False (off).
     exact_values : bool, optional
-        If True, interpolation provides input values at input locations. 
-        If False, interpolation accounts for variance/nugget within input 
-        values at input locations and does not behave as an 
+        If True, interpolation provides input values at input locations.
+        If False, interpolation accounts for variance/nugget within input
+        values at input locations and does not behave as an
         exact-interpolator [2]. Note that this only has an effect if
         there is variance/nugget present within the input data since it is
         interpreted as measurement error. If the nugget is zero, the kriged
         field will behave as an exact interpolator.
+    pseudo_inv : :class:`bool`, optional
+        Whether the kriging system is solved with the pseudo inverted
+        kriging matrix. If `True`, this leads to more numerical stability
+        and redundant points are averaged. But it can take more time.
+        Default: False
+    pseudo_inv_type : :class:`int`, optional
+        Here you can select the algorithm to compute the pseudo-inverse matrix:
+
+            * `1`: use `pinv` from `scipy` which uses `lstsq`
+            * `2`: use `pinv2` from `scipy` which uses `SVD`
+            * `3`: use `pinvh` from `scipy` which uses eigen-values
+
+        Default: `1`

```

Comment:
I think it would be more explicit for the user

```suggestion
    pseudo_inv_type : str, default='pinv'
        Here you can select the scipy algorithm to compute the pseudo-inverse matrix:

            * ``pinv`` uses `lstsq`
            * ``pinv2`` uses `SVD`
            * ``pinvh`` uses eigen-values
```

Reply:
Done.